runtime/libia2/include/ia2_internal.h

```diff
@@ -424,6 +424,8 @@ __attribute__((__noreturn__)) void ia2_reinit_stack_err(int i);
 #endif
 /* clang-format on */
 
+void create_thread_keys(void);
```

Comment:
nit: maybe namespace this since it's not static (i.e. `ia2_create_thread_keys`)

Reply:
Fixed in 1d36dae3c.